GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -158,7 +158,7 @@ GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
 }
 
 NSString *_Nonnull GDTCORDeviceModel() {
-  __block NSString *deviceModel = [UIDevice currentDevice].model;
+  __block NSString *deviceModel = @"";
```

Comment:
@"Unknown" might be a better alternative

Reply:
Done.